public/locales/en.json

```diff
@@ -53,6 +53,7 @@
     "Address": "Address",
     "Content": "Content",
     "Controller": "Controller",
+    "email": "Email",
```

Comment:
Please duplicate the same in `public/locals/en.json`.

You can just run `yarn i18n` to generate it

Reply:
Running `yarn i18n` seems to add the corresponding translation to `public/locals/cn.json` - This leaves a English translation in the `cn.json` file, is that ok? 

Results from runniny `yarn i18n`:
`public/locales/cn.json`
```json
...
    "edit": "编辑",
    "email": "Email",
    "expires": "到期",
...
```

I'm certainly not qualified to do an actual CN translation for the site. :-)

Reply:
yes, that's expected. It's more for the place holder.